ani-cli

```diff
@@ -84,8 +84,10 @@ update_script() {
 dep_ch() {
 	for dep; do
 		if ! command -v "$dep" >/dev/null; then
-			die "Program \"$dep\" not found. Please install it."
-		fi
+		    if flatpak info io.mpv.Mpv 2>&1 | grep -q 'error'; then
+			    die "Program \"$dep\" not found. Please install it."
+            fi
+        fi
```

Comment:
I don't understand the logic here, it appears to me that you check for flatpak mpv whenever any dependency is missing. This would be bad because if flatpak mpv is present, it would mask out a missing dependency, eg. grep.
I'm afraid flatpak programs need to be handled separately, if I understand this correctly

Reply:
oh, you are right.  Its a bad code